costmodel/router.go

```diff
@@ -920,7 +920,23 @@ func (a *Accesses) recordPrices() {
 			return strings.Split(key, ",")
 		}
 
+		// Create job to refresh provider prices every week
+		// TODO: Make configurable?
+		refreshProviderCosts := NewTimedJob(7*24*time.Hour, func() error {
```

Comment:
So if this fails, retry currently happens in a week? Seems fine for an initial implementation but just wanted to ask

Reply:
Yeah - it just doesn't feel like something we want to retry. I'm not sure what type of "failure" can happen outside of failing to download the sku list.